examples/renameimage.py

```diff
@@ -38,44 +97,100 @@
 
     glancesync = GlanceSync()
     glancesync.init_logs()
+    master = glancesync.master_region
+
+    master_images = glancesync.get_images_region(master)
+
+    old_image = None
+    new_image = None
+    for image in master_images:
+        if image.name == sys.argv[1]:
+            if old_image:
+                msg = 'There are more than an image with the old name in the' \
+                      ' master region'
+                logging.error(msg)
+                sys.exit(-1)
+            else:
+                old_image = image
+        elif image.name == sys.argv[2]:
+            if new_image:
+                msg = 'There are more than an image with the new name in the' \
+                      ' master region'
+                logging.error(msg)
+                sys.exit(-1)
+            else:
+                new_image = image
+
+    if old_image and new_image:
+        msg = 'Both an image with the new name and the old new exists in the' \
+            ' master region'
+        logging.error(msg)
+        sys.exit(-1)
+
+    if new_image:
+        master_image = new_image
+    elif old_image:
+        master_image = old_image
+        old_image.name = sys.argv[2]
```

Comment:
If I understand the algorithm, in this point you are taking the old_image.name the name of the second parameter that previously you are comparing for the new_image, it is something confuse.